.gitignore

```diff
@@ -0,0 +1,9 @@
+.idea/
+.vscode/
+*.iml
+.env
+.DS_Store
+venv/
+.pytest_cache/
+**__pycache__/
+db.sqlite3
```

Comment:
Add empty line

Reply:
the test check it for existing , 
but task does not demand it  existing,
so ,
its like  future template